tests/covalent_tests/workflow/electron_test.py

```diff
@@ -117,3 +117,32 @@ def workflow(x):
 
     assert list(g.nodes) == [0, 1]
     assert list(g.edges) == [(1, 0, 0)]
+
+
+    
+def test_metadata_in_electron_list():
+    """Test if metadata of the created electron list apart from executor is set to default values"""
+
+
+    def identity(y):
+        return y
+
+    calldep = ct.DepsCall(identity, args=[5], retval_keyword="y")
+
+    @ct.electron(call_before=[calldep])
+    def task(x, y=0):
+        return (x, y)
+
+    @ct.lattice
+    def workflow(x):
+        return task(x)
+
+    workflow.build_graph([2])
+    
+    task_metadata = workflow.transport_graph.get_node_value(0, "metadata")
+    e_list_metadata = workflow.transport_graph.get_node_value(1, "metadata")
+
+    assert list(e_list_metadata["call_before"]) == []
+    assert list(e_list_metadata["call_after"]) == []
+    
+    assert e_list_metadata["executor"] == task_metadata["executor"]
```

Comment:
To make the test a bit more comprehensive, could you set an executor instance for `task`, like
```python3
ct.electron(executor=LocalExecutor())
def task(x, y=0):
```

Reply:
Done.